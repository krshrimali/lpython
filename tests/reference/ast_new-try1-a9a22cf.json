{
    "basename": "ast_new-try1-a9a22cf",
    "cmd": "lpython --show-ast --new-parser --no-color {infile} -o {outfile}",
    "infile": "tests/parser/try1.py",
    "infile_hash": "266495ae838389f4f38ada4d482f7eb0ba2a8f405d9fd4dcdd007350",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "ast_new-try1-a9a22cf.stderr",
    "stderr_hash": "7742d2a2f790fafd824cf596210850b1368506bb2f64e2acde367a42",
    "returncode": 1
}