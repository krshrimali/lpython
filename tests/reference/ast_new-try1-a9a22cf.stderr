syntax error: Token ':' is unexpected here
 --> tests/parser/try1.py:1:25
  |
1 | for i in range(3, 0, -1):
  |                         ^ 
